project/Build.scala

```diff
@@ -12,7 +12,7 @@ object DottyBuild extends Build {
   val baseVersion = "0.1"
   val isNightly = sys.env.get("NIGHTLYBUILD") == Some("yes")
 
-  val jenkinsMemLimit = List("-Xmx1300m")
+  val jenkinsMemLimit = List("-Xmx1500m")
```

Comment:
The question still persists. Why does partest need substantially more memory to run?

I'm sure that previous September we were able to compile dotty & stdlib using only 750mb of memory without GC-related issues(i had to work on a very slow machine that had 2GB of memory for a week and I've decreased -Xmx to 750mb). In March, you  https://github.com/lampepfl/dotty/commit/acf05353a599de11e466b5e35cd003a2f6aa5312 have bumped the upper limit to 1.3Gb. Now we bump it to 1.5GB. 

Since last September, tests didn't grow much. Yes, dotty is now 20% bigger, white list is negligibly bigger. Tests do not explain why dotty now needs _twice_ more to pass the test suite, compared to situation 1 year ago.